### PR TITLE
Update machinecontroller to 1.2.1 and use proxy for kubelets

### PR DESCRIPTION
**Pull request by kron4eg**
```release-note
kubelets will have proxy support if it's specified in config
```

### CODE REVIEW

File: pkg/installer/installation/prerequisites.go [hunk 1]
```diff
@@ -26,7 +26,174 @@ import (
 	"github.com/kubermatic/kubeone/pkg/util"
 )
 
-const dockerVersion = "18.09.2"
+const (
+	dockerVersion = "18.09.6"
+)
+
+const (
```
Comment (xmudrii) [inline pkg/installer/installation/prerequisites.go]: Why do you repeat `const` two times?

Comment (kron4eg) [reply pkg/installer/installation/prerequisites.go]: it's two blocks of consts, 1 for "constants" and 1 for "scripts"

Comment (xmudrii) [reply pkg/installer/installation/prerequisites.go]: Usually, that doesn't matter at all, but I'm not going to block on this.

**Comment by kron4eg** [comment]
/override pull-kubeone-e2e-digitalocean-upgrade-1.13.5-1.14.1

**Pull request merged**